website/docs/r/automation_schedule.html.markdown

```diff
@@ -61,6 +65,22 @@ The following arguments are supported:
 
 * `timezone` - (Optional) The timezone of the start time. Defaults to `UTC`. For possible values see: https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
 
+* `advanced_schedule` - (Optional) Advanced fine-grained schedule frequency configuration. The `advanced_schedule` block supports fields documented below.
+
+The `advanced_schedule` block supports:
```

Comment:
_minor_ can we add `---` above this line (and then a blank line) to add a divider